src/Desktop.Analyzers/UnitTests/DoNotUseInsecureDTDProcessingUseXmlReaderForSchemaReadTests.cs

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
All the methods' signatures are referencing XmlReader instead of XmlTextReader, which is a sub class of XmlReader. So it is reasonable to say xmlReader. Not an issue.